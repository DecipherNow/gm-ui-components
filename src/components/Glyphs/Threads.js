import React from "react";
import Icon from "../Icon";

export default function Threads(props) {
  return (
    <Icon {...props} glyphName="Threads">
      <path
        d="M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z"
        id="shape"
      />
    </Icon>
  );
}

Threads.displayName = "Threads";
