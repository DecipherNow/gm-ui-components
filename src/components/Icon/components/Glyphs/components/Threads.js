import React from "react";

export default function Threads() {
  return (
    <path
      d="M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z"
      id="shape"
    />
  );
}
