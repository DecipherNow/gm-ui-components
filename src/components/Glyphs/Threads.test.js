import React from "react";
import { shallow } from "enzyme";
import Threads from "./Threads";

describe("Threads", () => {
  it("matches snapshot", () => {
    const aThreads = shallow(<Threads />);
    expect(aThreads).toMatchSnapshot();
  });
});
