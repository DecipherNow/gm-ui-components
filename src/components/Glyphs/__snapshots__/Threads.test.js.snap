// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Threads matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Threads />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "ariaLabelledby": "ariaLabelledby",
      "backgroundColor": "currentColor",
      "backgroundOpacity": 1,
      "backgroundSizeRatio": 1,
      "borderColor": "currentColor",
      "borderOpacity": 1,
      "borderWidth": 1,
      "children": <path
        d="M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z"
        id="shape"
      />,
      "glyph": "",
      "glyphColor": "currentColor",
      "glyphName": "Threads",
      "glyphSizeRatio": 1,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "d": "M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z",
        "id": "shape",
      },
      "ref": null,
      "rendered": null,
      "type": "path",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "ariaLabelledby": "ariaLabelledby",
        "backgroundColor": "currentColor",
        "backgroundOpacity": 1,
        "backgroundSizeRatio": 1,
        "borderColor": "currentColor",
        "borderOpacity": 1,
        "borderWidth": 1,
        "children": <path
          d="M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z"
          id="shape"
        />,
        "glyph": "",
        "glyphColor": "currentColor",
        "glyphName": "Threads",
        "glyphSizeRatio": 1,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "d": "M7,7 L19,7 L19,9 L7,9 L7,7 Z M6,11 L18,11 L18,13 L6,13 L6,11 Z M5,15 L17,15 L17,17 L5,17 L5,15 Z",
          "id": "shape",
        },
        "ref": null,
        "rendered": null,
        "type": "path",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
